packages/components/button/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@spark-ui/internal-utils": "^1.6.1",
     "@spark-ui/slot": "^1.5.2",
+    "@spark-ui/spinner": "^0.2.8",
```

Comment:
Unrelated to your PR, but it seems odd that the spinner package is still at version ~0.

Is it not considered stable yet?